inference-engine/cmake/templates/InferenceEngineConfig.cmake.in

```diff
@@ -19,150 +17,55 @@
 #   IE::inference_engine_c_api      - The Inference Engine C API library
 #
 
-macro(ext_message TRACE_LEVEL)
-    if (${TRACE_LEVEL} STREQUAL FATAL_ERROR)
-        if(InferenceEngine_FIND_REQUIRED)
-            message(FATAL_ERROR "${ARGN}")
-        elseif(NOT InferenceEngine_FIND_QUIETLY)
-            message(WARNING "${ARGN}")
-        endif()
-        return()
-    elseif(NOT InferenceEngine_FIND_QUIETLY)
-        message(${TRACE_LEVEL} "${ARGN}")
-    endif ()
-endmacro()
+@PACKAGE_INIT@
 
-set(InferenceEngine_FOUND FALSE)
+include(CMakeFindDependencyMacro)
 
-if(TARGET IE::inference_engine)
-    set(InferenceEngine_FOUND TRUE)
-    get_target_property(InferenceEngine_INCLUDE_DIRS IE::inference_engine INTERFACE_INCLUDE_DIRECTORIES)
-    set(InferenceEngine_LIBRARIES IE::inference_engine
-                                  IE::inference_engine_c_api)
-else()
-    if (WIN32)
-        set(_ARCH intel64)
-    else()
-        string(TOLOWER ${CMAKE_SYSTEM_PROCESSOR} _ARCH)
-        if(_ARCH STREQUAL "x86_64" OR _ARCH STREQUAL "amd64") # Windows detects Intel's 64-bit CPU as AMD64
-            set(_ARCH intel64)
-        elseif(_ARCH STREQUAL "i386")
-            set(_ARCH ia32)
-        endif()
-    endif()
+# need to store current PACKAGE_PREFIX_DIR, because it's overwritten by ngraph one
+set(IE_PACKAGE_PREFIX_DIR "${PACKAGE_PREFIX_DIR}")
 
-    set(THREADING "@THREADING@")
+set(THREADING "@THREADING@")
+if(THREADING STREQUAL "TBB" OR THREADING STREQUAL "TBB_AUTO")
+    set_and_check(_tbb_dir "@PACKAGE_IE_TBB_DIR@")
+    find_dependency(TBB
+                    COMPONENTS tbb tbbmalloc
```

Comment:
Looks like `COMPONENTS` argument did not work until cmake 3.9.0:
* https://gitlab.kitware.com/cmake/cmake/-/commit/ab358d6a859d8b7e257ed1e06ca000e097a32ef6
* https://cmake.org/pipermail/cmake-developers/2016-February/027876.html
* https://gitlab.kitware.com/cmake/cmake/-/merge_requests/872